conf/evolutions/44.js

```diff
@@ -0,0 +1,12 @@
+// Add the flag "tokenType = Authentication" to every token that is currently in the bearerTokenAuthenticator-collection
+
+// --- !Ups
+db.bearerTokenAuthenticators.find().forEach(function(elem) {
```

Comment:
forEach is always slow. I think in this instance we can use an update query with multi=true.

Reply:
then again, we won’t have a ton of tokens, so its probably fine here. but still, also for future reference